pkg/apis/core/validation/validation.go

```diff
@@ -3365,9 +3365,7 @@ func validateSchedulingGates(schedulingGates []core.PodSchedulingGate, fldPath *
 	// There should be no duplicates in the list of scheduling gates.
 	seen := sets.String{}
 	for i, schedulingGate := range schedulingGates {
-		if schedulingGate.Name == "" {
-			allErrs = append(allErrs, field.Required(fldPath.Index(i), "must not be empty"))
-		}
+		allErrs = append(allErrs, ValidateQualifiedName(schedulingGate.Name, fldPath.Index(i))...)
```

Comment:
Making this consistent with readiness gates makes sense to me

This validation is ok to change because this is still an alpha-gated field. This means we can promote to beta in 1.28, when the N-1 server agrees with the validation we are doing in the first beta release.

Reply:
we were targeting to release this to beta in 1.27, can we backport this to 1.26 and continue with our plans?

Reply:
@liggitt can we back-port this to 1.26? so it won't impact the schedule to graduate this feature to beta in 1.27.

Reply:
Thanks Jordan!